### PR TITLE
OSDOCS-4922 : Add local zones graphics to architecture models section.

### PR DESCRIPTION
**Pull request by eohartman**
Version(s):
4.12+
V1.2.10+

Issue:
[OSDOCS-4922](https://issues.redhat.com/browse/OSDOCS-4922)

Link to docs preview:
[ROSA architecture with local zones](https://64447--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_architecture_sub/rosa-architecture-models#rosa-architecture-local-zones_rosa-architecture-models)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by eohartman** [comment]
@arendej Will you please take a look at the preview linked above and let me know what you think? Thank you!

**Comment by eohartman** [comment]
@arendej  Will you please take a look at the preview linked above and let me know what you think? Thank you!

**Review by arendej — APPROVED** [review]
looks great

**Comment by eohartman** [comment]
@abhatt-rh Updated with content type. Thanks for catching that!

File: modules/rosa-architecture-local-zones.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa_architecture_sub/rosa-architecture-models.adoc
+:_content-type: <REFERENCE>
```
Comment (abhatt-rh) [inline modules/rosa-architecture-local-zones.adoc:4]:
```suggestion
:_content-type: REFERENCE  
```

**Pull request merged**